lib/proto/proto_conn.h

```diff
@@ -15,22 +15,23 @@ enum {
 };
 
 /**
- * proto_conn_create(s, sas, decr, nopfs, requirepfs, nokeepalive, K, timeo,
- *     callback_dead, cookie):
+ * proto_conn_create(s, sas, sa_b, decr, nopfs, requirepfs, nokeepalive, K,
+ *     timeo, callback_dead, cookie):
  * Create a connection with one end at ${s} and the other end connecting to
- * the target addresses ${sas}.  If ${decr} is 0, encrypt the outgoing data;
- * if ${decr} is nonzero, decrypt the incoming data.  If ${nopfs} is non-zero,
- * don't use perfect forward secrecy.  If ${requirepfs} is non-zero, drop
- * the connection if the other end tries to disable perfect forward secrecy.
- * Enable transport layer keep-alives (if applicable) on both sockets if and
- * only if ${nokeepalive} is zero.  Drop the connection if the handshake or
+ * the target addresses ${sas}.  Bind outgoing address to ${sa_b} if it is
+ * not NULL.  If ${decr} is 0, encrypt the outgoing data;  if ${decr} is
```

Comment:
There's two spaces in "data;  if".  That should only be 1 space, please.

Reply:
Fixed.